Sem1HW/Task8.py

```diff
@@ -0,0 +1,13 @@
+# Задача №8. 
+# Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек,
+# если разрешается сделать один разлом по прямой между дольками 
+# (то есть разломить шоколадку на два прямоугольника).
+
+n = int(input("Введите кол-во долек N: "))
+m = int(input("Введите кол-во долек M: "))
+k = int(input("Введите кол-во долек K: "))
+
+if (k % m == 0 or k % n == 0) and k < (n * m) - n:
```

Comment:
Тут явно что-то лишнее. Разрешите я поправлю, а вы подумаете, почему я это сделал?
```suggestion
if (k % m == 0 or k % n == 0) and k < n * m:
```